modules/kubernetes-addons/main.tf

```diff
@@ -255,6 +255,18 @@ module "ondat" {
   admin_password    = var.ondat_admin_password
 }
 
+module "kube_prometheus_stack" {
+  count       = var.enable_kube_prometheus_stack ? 1 : 0
+  source      = "./kube-prometheus-stack"
+  helm_config = var.kube_prometheus_stack_helm_config
+  #AWS Managed Prometheus Workspace
```

Comment:
Do we want to remove these commented attributes for Amazon Managed Service for Prometheus?

Reply:
yes though i think the [values file allows configuration of the remote write](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L2533) but i didn't get to testing that yet. We should be able to configure this similar to the existing prometheus add-on later.